vimeo-networking/src/main/java/com/vimeo/networking/model/VideoFile.java

```diff
@@ -88,6 +88,9 @@ public String toString() {
     private VideoQuality quality;
     private MimeType type;
     public VideoLog log;
+    /** The md5 provides us with a way to uniquely identify video files at {@link #link} */
+    @SerializedName("md5")
+    public String md5;
```

Comment:
Why do you need a ```SerializedNamed``` annotation if it's the same as the variable name? Doesn't hurt to have it, but we don't have it elsewhere.